Rakefile

```diff
@@ -4,9 +4,19 @@ require 'bundler'
 require 'bundler/setup'
 require 'rspec/core/rake_task'
 
+EXAMPLE_APP_BUNDLE_PATH = File.expand_path('../vendor/example_app-bundle', __FILE__)
+EXAMPLE_APP_HOME = 'spec/resources/example_app'
```

Comment:
If it isn't a problem to have both paths absolute, I think that would be less confusing